cpmpy/expressions/globalconstraints.py

```diff
@@ -468,7 +464,8 @@ def decompose(self):
                     demand_at_t += demand * ((start[job] <= t) & (t < end[job]))
                 else:
                     demand_at_t += demand[job] * ((start[job] <= t) & (t < end[job]))
-            cons += [capacity >= demand_at_t]
+
```

Comment:
why does the order matter here?

Reply:
I'm not sure __exactly__ why it matters but in some cases the comparison was not evaluated and just become "True"... I think it has something to do if `capacity` is a numpy something vs just Python int... Anyway this seems to fix it in all cases. I will add a test where the old version failed still

Reply:
Ok so it is indeed the Numpy thingy...
I will add a test to ensure we do not encounter this again